src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -397,6 +431,8 @@ private OAuthAccessGrant lookupGrant(
         private String subject;
         private long authorization;
         private long expiresAt;
+        private long challengeResponseTimeout;
+        private long challengedAt;
```

Comment:
Since `challengeAt` and `challengedAt` are very similar with quite different meanings, suggest renaming this `lastChallengedAt` instead.